UE4SS/src/GUI/LiveView.cpp

```diff
@@ -177,6 +177,23 @@ namespace RC::GUI
         }
     }
 
+    static void attempt_to_add_search_by_address_result(uintptr_t address_to_search_by, UObject* object)
+    {
+        auto object_addr = std::bit_cast<uintptr_t>(object);
+        if (address_to_search_by < object_addr)
+        {
+            return;
+        }
+
+        auto uclass = object->IsA<UStruct>() ? static_cast<UClass*>(object) : object->GetClassPrivate();
+        uintptr_t end_addr = object_addr + uclass->GetPropertiesSize();
+        //Output::send<LogLevel::Default>(STR("object_addr {:016X}-{:016X}"), object_addr, end_addr);
+        if (address_to_search_by < end_addr) {
```

Comment:
```suggestion
        if (address_to_search_by <= end_addr) {
```
Shouldn't this be inclusive ? If the address matches exactly then the result is probably still something that the user is interested in.

Reply:
There could be some extra fields not reported in the size, or padding. I don't know much about unreal internals. In my testing there was some padding, but I don't know how consistent that is.
`end_addr` is probably misnamed. If it were in a C array, and `GetPropertiesSize()` is accurate, then address+size = the address of the next object.